test/replication.test.js

```diff
@@ -1803,4 +1908,36 @@ describe('Replication / Change APIs', function() {
   function getIds(list) {
     return getPropValue(list, 'id');
   }
+
+  function assertTargetModelEqualsSourceModel(conflicts, sourceModel,
```

Comment:
AFAICT, this helper is defined on L289 too ([source](https://github.com/kobaska/loopback/blob/b1fdd53dbc433c7ecadfeef1cf31f3d6fe61f3e2/test/replication.test.js#L298)). What is the difference between these two variants of the helper? Is there any reason prevent us from having only a single shared copy of this function?